Gemfile

```diff
@@ -47,7 +47,7 @@ group :development, :test do
   gem 'rspec-activemodel-mocks'
   gem 'factory_girl_rails'
   gem 'database_cleaner'
-  gem 'byebug'
+  gem 'byebug', require: !ENV['RM_INFO']
```

Comment:
@F3PiX Could you add a tiny comment at the end of the line that it is RubyMine-related?